lld/test/lit.cfg.py

```diff
@@ -36,6 +36,7 @@
 
 llvm_config.use_default_substitutions()
 llvm_config.use_lld()
+config.substitutions.append(("%llvm_src_root", config.llvm_src_root))
```

Comment:
I grepped this variable.

clang does it in a different way. Do you know the difference?
```
clang/test/lit.site.cfg.py.in
5:config.llvm_src_root = path(r"@LLVM_SOURCE_DIR@")
```